# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/nsBaseWidget.h
# Commit: 3dd35da1c8be
# Full Hash: 3dd35da1c8bee1353c935e045e7518bc01ea3e77
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-05-09 21:30:55
# Description:
#   Bug 1635769 - Call WebRenderLayerManager::ClearCachedResources() during re-creating widget r=tnikkel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74392
# ==============================================================================

diff -r efb86c809f5c -r 3dd35da1c8be widget/nsBaseWidget.h
--- a/widget/nsBaseWidget.h	Sat May 09 09:38:52 2020 +0300
+++ b/widget/nsBaseWidget.h	Sat May 09 05:44:37 2020 +0000
@@ -578,6 +578,8 @@
 
   virtual CompositorBridgeChild* GetRemoteRenderer() override;
 
+  virtual void ClearCachedWebrenderResources() override;
+
   /**
    * Notify the widget that this window is being used with OMTC.
    */